lib/assets/javascripts/views/questions/sharedEventHandlers.js

```diff
@@ -0,0 +1,32 @@
+const onChangeQuestionFormat = (e) => {
+  const source = e.target;
+  const selected = source.value;
+  const defaultValue = $(source).closest('form').find('[data-attribute="default_value"]');
+  const questionOptions = $(source).closest('form').find('[data-attribute="question_options"]');
+  switch (selected) {
+    case '1':
```

Comment:
Would be good to add comments here at some point to indicate what question type each case is

Reply:
Totally agree. We would need to have it commented and probably is best to have constants in place...